controller/status.go

```diff
@@ -213,6 +213,13 @@ var (
 	StatusIndex = []StatusContext{
 		{
 			FleetCurrent:  "inactive",
+			FleetDesired:  "launched",
+			SystemdActive: "active",
+			SystemdSub:    "running",
+			Aggregated:    StatusRunning,
+		},
```

Comment:
This is the actual fix - before, the `StatusContext` beneath was matched for this case, which lead `inago` to believe the aggregated status was `StatusStopped`, which was incorrect. This lead to repeatedly waiting for the status to become running.